Subject: Linux-RT 2.6.25.4-RT
From: http://www.kernel.org/pub/linux/kernel/projects/rt/
Acked-by: Sven-Thorsten Dietrich <sdietrich@suse.de>
This patch fixes this compile error in 2.6.24.7-rt8:
 CC      arch/arm/kernel/process.o
arch/arm/kernel/process.c: In function 'cpu_idle':
arch/arm/kernel/process.c:175: error: implicit declaration of function 'trace_preempt_exit_idle'
arch/arm/kernel/process.c:180: error: implicit declaration of function 'trace_preempt_enter_idle'

Signed-off-by: Remy Bohmer <linux@bohmer.net>
---
 arch/arm/kernel/process.c |    2 --
 1 file changed, 2 deletions(-)

Index: linux-2.6.25.4-rt4/arch/arm/kernel/process.c
===================================================================
--- linux-2.6.25.4-rt4.orig/arch/arm/kernel/process.c	2008-05-29 09:46:35.000000000 -0400
+++ linux-2.6.25.4-rt4/arch/arm/kernel/process.c	2008-05-29 09:47:27.000000000 -0400
@@ -171,12 +171,10 @@ void cpu_idle(void)
 			idle();
 		leds_event(led_idle_end);
 		local_irq_disable();
-		trace_preempt_exit_idle();
 		tick_nohz_restart_sched_tick();
 		__preempt_enable_no_resched();
 		__schedule();
 		preempt_disable();
-		trace_preempt_enter_idle();
 		local_irq_enable();
 	}
 }
